### PR TITLE
CFIN-390 Local performance testing scripts

### PR DESCRIPTION
**Pull request by ChristianBeddows**
Please check this out locally and have a crack at using the npm scripts. You'll need to be running a production server on port 3000 prior to the run for this to work.

### CODE REVIEW

**Comment by ncn1** [comment]
> You'll need to be running a production server on port 3000 prior to the run for this to work.

ie. run the 'prod' npm script

**Review by FergusMcGlynn — APPROVED** [review]
I've run this locally and it seems to work fine for both desktop and mobile. I get really bad scores for desktop, but that's another ticket.

**Review by ncn1 — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
   "private": true,
   "scripts": {
     "dev": "next dev",
+    "prod": "npm run build && npm run start",
```
Comment (ce666-york) [inline package.json:7]: Does this use development or production environment variables? Wondering if it's slightly misnamed and better to just roll into `start`

Comment (FergusMcGlynn) [reply package.json:7]: It'll be running production environment variables

Comment (ce666-york) [reply package.json:7]: Thanks Ferg. I think in my head there's a slightly separate (and wider) issue of local environment variables, when you might expect to use them, and what our defaults are. This also ties into the question of ideally how would we want to develop locally, as currently there's a dependency on the development or production API.

**Review by ce666-york — CHANGES_REQUESTED** [review]
These scripts seem to give significantly worse performance scores for both desktop (mid 60's) and mobile (mid 30's) compared to what running Lighthouse in Chrome/Pagespeed insights give (mid 90's for desktop and mid 60's for mobile).

I think we need to work out why this is the case before we can use these scripts - looks great so far but I think there's a bit more to do!

**Review by ce666-york — APPROVED** [review]
(Should have marked as approved as in principle I think these changes are fine to be merged!)

**Review by ce666-york — CHANGES_REQUESTED** [review]
Actually I think this PR should be for merging into the story branch `CFIN-140` rather than `dev`. It'll probably be good to merge `CFIN-140` back into your branch too so you can develop against the build with the performance improvements.

**Review by mrgibbonsuoy — APPROVED** [review]

**Comment by ChristianBeddows** [comment]
I'm closing this PR as I need to do some git tidying, and further changes to get the script working at an acceptable level.

**Pull request closed without merging**